src/server/pps/persist/server/rethink_api_server.go

```diff
@@ -137,16 +137,15 @@ func (a *rethinkAPIServer) Close() error {
 // Timestamp cannot be set
 func (a *rethinkAPIServer) CreateJobInfo(ctx context.Context, request *persist.JobInfo) (response *persist.JobInfo, err error) {
 	defer func(start time.Time) { a.Log(request, response, err, time.Since(start)) }(time.Now())
-	if request.JobID != "" {
-		return nil, fmt.Errorf("request.JobID should be unset")
-	}
 	if request.CreatedAt != nil {
```

Comment:
Did we need to add the ability to create jobs with the JobID already set?
Why was that?